### ISSUE TITLE
Incorrect filesize reporting on very large files

### ISSUE THREAD

**Issue by meoberto** [jfurrow/flood#504]
I have reached past the TB of upload in two torrents, however, as we can see here, the uploaded data size is displayed incorrectly, something which is evident in the Flood screenshot below, where I sorted my torrents by descending uploaded data.
![screenshot_20171025_101527](https://user-images.githubusercontent.com/30121325/31987927-a8fec6f4-b96d-11e7-84d1-93832270c25d.png)

On these top two torrents, rTorrent does however report the uploaded data correctly
![screenshot_20171025_101252](https://user-images.githubusercontent.com/30121325/31987954-b90c0d40-b96d-11e7-9514-c3b547dd6dd3.png)

Basically TB file sizes are being displayed as GB.
